doc/header.html.in

```diff
@@ -21,7 +21,8 @@
     <body>
         <div id="banner-container">
             <div id="banner">
-                <span id="sfml">SFML @VERSION_MAJOR@.@VERSION_MINOR@.@VERSION_PATCH@</span>
+                <span id="sfml">SFML @PROJECT_VERSION@</span>
```

Comment:
Wait, it looks like this CMake variable not defined. That's because we do not define the version with the `project` command.

Reply:
woops